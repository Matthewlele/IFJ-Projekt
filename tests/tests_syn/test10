x = hchkrdtn"

if x == "hchkrdtn" then
  print "Tvrde samohlasky"
end
