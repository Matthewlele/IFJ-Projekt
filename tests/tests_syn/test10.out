exptected SCANNER ERR: row 1
